errata.md

```diff
@@ -1,32 +1,71 @@
 # Errata
 
 The following are errata for the Metamath 2019 ("Second Edition") book
-dated 2019-06-02:
+dated 2019-06-02. Issues marked with an asterisk (*) have not yet been
+fixed in the repository version of the book.
 
+* Preface (page x) - "able follow a proof" should be "able to follow a proof".
+* Preface (page xiv, Note on Bibliography and Index) - "find it in on" should
+  be "find it on".
 * Preface (Note Added March 7, 2019) - Matamath --> Metamath
-* Section 1.4.2 (page 32, paragraph 3) -
+* Section 1.1.1 (page 4) - "allows you do" should be "allows you to do".
+* Section 1.1.6 (page 20) - "may be correspond" should be "may correspond".
+* Section 1.3.2 (page 27) - "a way that can not only able to" should be "a way
+  that is not only able to".
+* Section 1.3.4 (page 28) - "is a mostly a proof verifier" should be "is mostly
+  a proof verifier".
+* *Section 1.4.2 (page 32, paragraph 3) -
   there's an extraneous ")" after the term set.mm.
-* Section 2.4 (page 52) and further shows this text after some assign commands:
+* Section 1.4.4 (page 34) - "making it is easy" should be "making it easy".
+* Section 2.3 (page 49) - "You can could use" should be "You could use".
+* *Section 2.4 (page 52) and further shows this text after some assign commands:
   "To undo the assignment, DELETE STEP ... and INITIALIZE, UNIFY if needed."
   This text is no longer shown.
   When the first edition of the book was written, metamath.exe didn't have
   the 'undo'/'redo' commands (added in 2013; see 'help undo'), so hints
   were provided by some commands for how to undo them manually.
   These hints are no longer displayed since they are no longer needed.
-* Section 3.3.2 and 3.3.3 headings (page 71) - the headings go past
+* Section 2.4 (page 53) - "examples where is it used" should be "examples
+  where it is used".
+* Section 3.1 (page 60) - "allow us determine" should be "allow us to
+  determine". In the same paragraph, "allows to say" should be "allows us to
+  say".
+* Section 3.2.2 (page 67) - "auxilliary" should be "auxiliary".
+* Section 3.2.4 (page 70) - "Tarsky-Grothendieck" should be
+  "Tarski-Grothendieck".
```

Comment:
It should also be an en-dash ("--" in LaTeX).

Reply:
[Wikipedia](https://en.wikipedia.org/wiki/Dash#Relationships_and_connections) says it's Ok either way:

> A distinction is often made between "simple" attributive compounds (written with a hyphen) and other subtypes (written with an en dash); at least one authority considers name pairs, where the paired elements carry equal weight, as in the [Taft–Hartley Act](https://en.wikipedia.org/wiki/Taft%E2%80%93Hartley_Act) to be "simple", while others consider an en dash appropriate in instances such as these to represent the parallel relationship, as in the [McCain–Feingold bill](https://en.wikipedia.org/wiki/Bipartisan_Campaign_Reform_Act) or [Bose–Einstein statistics](https://en.wikipedia.org/wiki/Bose%E2%80%93Einstein_statistics). When an act of the U.S. Congress is named using the surnames of the senator and representative who sponsored it, the hyphen-minus is used in the [short title](https://en.wikipedia.org/wiki/Short_title); thus the short title of [Public Law 111–203](https://en.wikipedia.org/wiki/Dodd%E2%80%93Frank_Wall_Street_Reform_and_Consumer_Protection_Act) is "The Dodd-Frank Wall Street Reform and Consumer Protection Act", with a hyphen-minus rather than an en dash between "Dodd" and "Frank". However, there is a difference between something named for a parallel/coordinate relationship between two people (for example, [Satyendra Nath Bose](https://en.wikipedia.org/wiki/Satyendra_Nath_Bose) and [Albert Einstein](https://en.wikipedia.org/wiki/Albert_Einstein)) and something named for a single person who had a [compound surname](https://en.wikipedia.org/wiki/Compound_surname), which may be written with a hyphen or a space but not an en dash (for example, the [Lennard-Jones potential](https://en.wikipedia.org/wiki/Lennard-Jones_potential) [hyphen] is named after one person (Mr. [John Lennard-Jones](https://en.wikipedia.org/wiki/John_Lennard-Jones)), as are [Bence Jones proteins](https://en.wikipedia.org/wiki/Bence_Jones_protein) and [Hughlings Jackson syndrome](https://en.wikipedia.org/wiki/John_Hughlings_Jackson)). Copyeditors use dictionaries (general, medical, biographical, and geographical) to confirm the [eponymity](https://en.wikipedia.org/wiki/Eponym) (and thus the styling) for specific terms, given that no one can know them all offhand.
>
> Preference for an en dash instead of a hyphen in these coordinate/relationship/connection types of terms is a matter of style, not inherent orthographic "correctness"; both are equally "correct", and each is the preferred style in some style guides. For example, [the American Heritage Dictionary of the English Language](https://en.wikipedia.org/wiki/The_American_Heritage_Dictionary_of_the_English_Language), the [AMA Manual of Style](https://en.wikipedia.org/wiki/AMA_Manual_of_Style), and [Dorland's medical reference works](https://en.wikipedia.org/wiki/Dorland%27s_medical_reference_works) use hyphens, not en dashes, in coordinate terms (such as "[blood-brain barrier](https://en.wikipedia.org/wiki/Blood%E2%80%93brain_barrier)"), in [eponyms](https://en.wikipedia.org/wiki/Eponym) (such as "[Cheyne-Stokes respiration](https://en.wikipedia.org/wiki/Cheyne%E2%80%93Stokes_respiration)", "[Kaplan-Meier method](https://en.wikipedia.org/wiki/Kaplan%E2%80%93Meier_estimator)"), and so on.